### DISCUSSION TITLE
commandline: Unknown option “--is-valid”

### DISCUSSION THREAD

**Discussion by DasDNS** [JanDeDobbeleer/oh-my-posh#6046] [Troubleshoot] [answered]
I'm using Ubuntu 22.04. I recently tried oh-my-posh-themes. I'm getting this error for the fish shell.

commandline: Unknown option “--is-valid”

- (line 195): 
    if commandline --is-valid || test -z (commandline --current-buffer | string trim -l | string collect)
       ^
in function '_omp_enter_key_handler'

(Type 'help commandline' for related documentation)

![Screenshot from 2024-12-22 22-37-40](https://github.com/user-attachments/assets/304b1474-4022-44fe-8760-3035f04b24a8)
![Screenshot from 2024-12-22 22-38-21](https://github.com/user-attachments/assets/b97209cb-ea62-47b0-8b2a-9e91fd5971ff)

**Comment by JanDeDobbeleer** [comment]
@DasDNS update fish, it's a bit outdated.

**Comment by DasDNS** [comment]
Updated both oh-my-posh and fish, but the error is still there.
![Screenshot from 2024-12-22 23-06-33](https://github.com/user-attachments/assets/d41dcb7c-779a-42c8-8f7c-85de46ece74c)

**Reply by JanDeDobbeleer** [reply, marked as answer]
@DasDNS that's not the latest version, you need `3.7.1`. Fish `3.3.1` is from July 6, 2021.

**Reply by DasDNS** [reply]
Yes, it is working without any issues now. Thank you very much for helping.
